external storage url in tidb cloud (#21058)
pingcap/docs: external-storage-uri.md

    --config=cdc_csv.toml
```

</CustomContent>

<CustomContent platform="tidb-cloud">

- `scheme`: `s3`
- `host`: `bucket name`
- `parameters`:

    - `access-key`: Specifies the access key.
    - `secret-access-key`: Specifies the secret access key.
    - `session-token`: Specifies the temporary session token. TiDB supports this parameter starting from v7.6.0.
    - `use-accelerate-endpoint`: Specifies whether to use the accelerate endpoint on Amazon S3 (defaults to `false`).
    - `endpoint`: Specifies the URL of custom endpoint for S3-compatible services (for example, `<https://s3.example.com/>`).
    - `force-path-style`: Use path style access rather than virtual hosted style access (defaults to `true`).
    - `storage-class`: Specifies the storage class of the uploaded objects (for example, `STANDARD` or `STANDARD_IA`).
    - `sse`: Specifies the server-side encryption algorithm used to encrypt the uploaded objects (value options: empty, `AES256`, or `aws:kms`).
    - `sse-kms-key-id`: Specifies the KMS ID if `sse` is set to `aws:kms`.
    - `acl`: Specifies the canned ACL of the uploaded objects (for example, `private` or `authenticated-read`).

>     - `role-arn`: When you need to access Amazon S3 data from a third party using a specified [IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html), you can specify the corresponding [Amazon Resource Name (ARN)](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) of the IAM role with the `role-arn` URL query parameter, such as `arn:aws:iam::888888888888:role/my-role`. For more information about using an IAM role to access Amazon S3 data from TiDB Cloud, see [Configure External Storage Access for TiDB Cloud Dedicated](/config-s3-and-gcs-access.md). TiDB supports this parameter starting from v7.6.0.

Reviewer: ```suggestion
    - `role-arn`: To access Amazon S3 data from a third party using a specified [IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html), you can specify the corresponding [Amazon Resource Name (ARN)](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) of the IAM role with the `role-arn` URL query parameter, such as `arn:aws:iam::888888888888:role/my-role`. For more information about using an IAM role to access Amazon S3 data from TiDB Cloud, see [Configure Amazon S3 access using a Role ARN](/tidb-cloud/config-s3-and-gcs-access.md#configure-amazon-s3-access-using-a-role-arn). For backup and restore, this parameter is supported starting from v7.6.0.
```

>     - `external-id`: When you access Amazon S3 data from TiDB Cloud, you must specify a correct [external ID](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html) to assume [the IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html). In this case, you must use this `external-id` URL query parameter to specify the external ID and make sure that you can assume the IAM role.

Reviewer: ```suggestion
    - `external-id`: To access Amazon S3 data from TiDB Cloud, you must specify the TiDB Cloud External ID with the `external-id` parameter to assume [the IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html). To get the TiDB Cloud External ID, see [Configure Amazon S3 access using a Role ARN](/tidb-cloud/config-s3-and-gcs-access.md#configure-amazon-s3-access-using-a-role-arn).
```
